### PR TITLE
Modify:remove 'connect to our backend' placeholder

### LINKED ISSUE #73: Remove unnecessary placeholders in the extension pop up page
### Summary

No logic has been implemented for the "Connect to our backend" placeholder section, so this placeholder should be removed.

### Basic Example

The extension will look the same minus the "Connect to our backend" section.

### Drawbacks

The extension might look more incomplete or like it has fewer features. I think having a simpler design is better than having "fake" features.

### Unresolved questions
It is not clear what connecting to the backend should do or how to do it. If this is clarified, a feature request can be made for it.

### Screenshot
截屏2022-09-21 下午6 13 48

### PR DESCRIPTION
**Pull request by Kanelmis**
## Motivation
No logic has been implemented for the "Connect to our backend" placeholder section, so this placeholder should be removed.

## Summary of changes
Remove unnecessary placeholders and related CSS configuration in the extension `popup.html` and `popup.css`.

## Attached GitHub issue
Closes #73 

### Documentation
- [x] Updated the readme documents (`README.md` etc.)
- [x] New functions and methods have additional comments added to document their usage

### Local Build
- [x] Ran all pre-commit hooks `pre-commit run --all-files`
- [x] Extension has been tested to build correctly `cd extension && yarn && yarn build`
- [x] Backend has been tested to build correctly `cd backend && docker-compose build`
- [x] Extension test suite has been run locally `cd extension && yarn && yarn test`
- [x] Backend test suite has been run locally `cd backend && python3 -m unittest`

### GitHub
- [x] Target branch has been set correctly
- [x] PR has been rebased onto target branch
- [x] PR has been assigned an owner
- [x] Author has performed a self-review of the code
- [x] Removed the WIP message from the top of this PR

### Screenshot
截屏2022-09-22 下午3 13 18

### CODE REVIEW

**Review by Destiny-02 — CHANGES_REQUESTED** [review]
Did you forget to commit the changes in `popup.html`? I can see the CSS changes but the text and toggle is still there. 
image

**Comment by Destiny-02** [comment]
Please use "Closes" keywords in the PR description so GitHub automatically closes the issue when the PR is merged.

**Comment by Destiny-02** [comment]
Please use the correct file names and code formatting in the PR description to help with readability e.g. `popup.html` and `popup.css`

**Review by Destiny-02 — CHANGES_REQUESTED** [review]
As you can see from https://github.com/se310-t6/politicry/pull/77/files. Only the CSS appears to have changed. I think you may have merged the last PR incorrectly which overrode your changes in `popup.html`. 

Thanks for using the correct file names in the PR descriptions. Please use the code formatting as well e.g. `popup.html` and `popup.css`. This can be done by wrapping the text with backticks `\.

**Comment by Kanelmis** [comment]
> Member

Yes. Somehow, the change has been discarded when resolving the conflict with the last commit.

File: extension/public/popup.html [hunk 1]
```diff
@@ -74,23 +74,6 @@ <h3>Keywords</h3>
                 </div>
             </div>
         </div>
-        <div class="bottom">
-            <div class="backend">
-                <span>Connect to our backend</span>
-                <label class="switch tooltip">
-                    <input type="checkbox" checked>
-                    <span class="slider round"></span>
-                    <span class="tooltiptext">
```
Comment (Destiny-02) [inline extension/public/popup.html:83]: `tooltiptext` is no longer used and can be removed from the CSS (unless it's used anywhere else?).

**Review by Destiny-02 — APPROVED** [review]

**Review by Destiny-02 — CHANGES_REQUESTED** [review]
See latest comment

File: extension/public/popup.html (hunk 1 above)
Comment (Destiny-02) [reply extension/public/popup.html:83]: Sorry this PR is not yet approved (I clicked the wrong button!)

Comment (Kanelmis) [reply extension/public/popup.html:83]: Yeah, you are correct. I first thought that the slider had been used by other sections and remained it in the CSS file. However, I then forgot to delete tooltips, lol

**Review by Destiny-02 — APPROVED** [review]
LGTM. Thanks for fixing up everything :)

**Pull request merged**